Sources/LinuxPlatform/Linux.swift

```diff
@@ -481,98 +535,62 @@ public struct Linux: Platform {
             } else {
                 print(message)
             }
-            return self.manualSelectPlatform(platformPretty)
+            return await self.manualSelectPlatform(platformPretty)
         }
 
-        let data = FileManager.default.contents(atPath: releaseFile)
-        guard let data = data else {
-            let message = "Unable to read OS release information from file \(releaseFile)"
-            if disableConfirmation {
-                throw Error(message: message)
-            } else {
-                print(message)
-            }
-            return self.manualSelectPlatform(platformPretty)
-        }
-
-        guard let releaseInfo = String(data: data, encoding: .utf8) else {
-            let message = "Unable to read OS release information from file \(releaseFile)"
-            if disableConfirmation {
-                throw Error(message: message)
-            } else {
-                print(message)
-            }
-            return self.manualSelectPlatform(platformPretty)
-        }
+        let releaseInfo = try String(contentsOfFile: releaseFile, encoding: .utf8)
 
         var id: String?
         var idlike: String?
         var versionID: String?
-        var ubuntuCodeName: String?
         for info in releaseInfo.split(separator: "\n").map(String.init) {
             if info.hasPrefix("ID=") {
                 id = String(info.dropFirst("ID=".count)).replacingOccurrences(of: "\"", with: "")
             } else if info.hasPrefix("ID_LIKE=") {
                 idlike = String(info.dropFirst("ID_LIKE=".count)).replacingOccurrences(of: "\"", with: "")
             } else if info.hasPrefix("VERSION_ID=") {
-                versionID = String(info.dropFirst("VERSION_ID=".count)).replacingOccurrences(of: "\"", with: "")
-            } else if info.hasPrefix("UBUNTU_CODENAME=") {
-                ubuntuCodeName = String(info.dropFirst("UBUNTU_CODENAME=".count)).replacingOccurrences(of: "\"", with: "")
```

Comment:
Do we still need to account for stings which start with "UBUNTU_CODENAME" here?

Reply:
We're reading this file line-by-line and we don't have any need for the ubuntu codename anymore. I think it's safe to just skip ubuntu codename lines.